Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:14.04
+MAINTAINER Yi Wang <yi.wang.2005@gmail.com>
+
+COPY build.sh /opt
```

Comment:
Not sure, but I think it does not cause data transferring, I have mounted pretty big folder but does not see launch speed slow down. That data transfer is in build process, this is while running docker. Hopefully not :)